PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -455,6 +455,9 @@
 		     doc="index of first subjet"),
         subJetIdx2 = Var("?nSubjetCollections()>0 && subjets('SoftDropPuppi').size()>1?subjets('SoftDropPuppi')[1].key():-1", int,
 		     doc="index of second subjet"),
+        nBHadrons=Var("jetFlavourInfo().getbHadrons().size()", int, doc="number of b-hadrons",precision=10),
```

Comment:
precision makes no sense for integers, please remove it and use uint8 (see elsewhere for an example) to save space - I guess we don't have more than 256 of these

Reply:
yes - done.